pkg/utils/utils_test.go

```diff
@@ -46,6 +46,11 @@ var _ = Describe("Utils", func() {
 		Expect(version).To(Equal(false))
 	})
 
+	It("check Docker version constrains", func() {
```

Comment:
Can you add a test that tests the env variable?

Reply:
@tczekajlo do you have any suggestions on preferred way of testing the env variable The method `checkDockerVersionConstrains()` where we check the env var is called upon Docker client instantiation. If we want to test the env var, we will need a mock Docker Client, right?

Reply:
Probably we can pull the env var check code out of `pkg/docker/client.go` as a standalone method in `pkg/docker/utils.go`, which is a lot easier for testing.

Reply:
Implemented the idea above & added unit tests 😛